docs/publishers/events.rst

```diff
@@ -0,0 +1,48 @@
+Custom DOM events
+=================
+
+Hypothesis client emits a few custom DOM events to which your app can react.
+
+.. option:: hypothesis:layoutchange
+
+   This event is emitted when the sidebar layout changes. Changes to the layout occur on load, when the sidebar is
+   toggled to show and hide, and when the user adjusts the sidebar manually.
+
+   The event will wrap some properties that can be used to tell the sidebar status and size.
```

Comment:
This sentence seems unnecessary to me. I think the documentation that follows is self-explanatory enough. A heading like "Properties" would make more sense, plus a note that events extend `CustomEvent` and properties are available on `detail`.